tests/mbt/src/driver.rs

```diff
@@ -63,476 +65,27 @@ impl Driver for EmeraldDriver {
             StartedRoundAction(node, height, round, proposer) => {
                 self.handle_started_round(node, height, round, proposer)
             },
-            GetValueAction(node, height, round) => {
-                self.handle_get_value(node, height, round)
-            },
-            ReceivedProposalAction(node, from, proposal) => {
-                self.handle_received_proposal(node, from, proposal)
+            GetValueAction(node, height, round, proposal) => {
+                self.handle_get_value(node, height, round, proposal)
             },
-            DecidedMessageAction(node, height, round, value_id) => {
-                self.handle_decided(node, height, round, value_id)
-            },
-            ProcessSyncedValueAction(node, height, round, proposer, payload) => {
-                self.handle_process_synced_value(node, height, round, proposer, payload)
-            },
-            GetDecidedValueAction(node, from, height) => {
-                self.handle_get_decided_value(node, from, height)
-            },
-            GetHistoryMinHeightAction(node, from) => {
-                self.handle_get_history_min_height(node, from)
+            ReceivedProposalAction(node, proposal) => {
+                self.handle_received_proposal(node, proposal)
             },
```

Comment:
Two questions here:
1. Is the plan to add this sync-related actions later for the bug-catching experiment? 2. Would a test with `quint run` here break and tell us to add these missing actions?